products/sle/main.pm

```diff
@@ -647,6 +647,9 @@ sub load_consoletests() {
             loadtest "rt/kmp_modules";
         }
         loadtest "console/consoletest_setup";
+        if (get_var('SCC_REGISTER', '') eq 'installation' && check_var("DESKTOP", "gnome")) {
+            loadtest "feature/feature_console/deregister";
```

Comment:
this probadly kills QAM tests,  please more specific loader + right place will be almost on end of tests